src/utils/Log.h

```diff
@@ -0,0 +1,53 @@
+/*
+ *  Copyright (C) 2020 Team Kodi (https://kodi.tv)
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSE.md for more information.
+ */
+
+#pragma once
+
+#include <kodi/General.h>
+
+namespace rtmpstream
+{
+
+inline const char* kodiTranslateLogLevel(const AddonLog logLevel)
+{
+  switch (logLevel)
+  {
+    case ADDON_LOG_DEBUG:
+      return "LOG_DEBUG:   ";
+    case ADDON_LOG_INFO:
+      return "LOG_INFO:    ";
+    case ADDON_LOG_NOTICE:
+      return "LOG_NOTICE:  ";
+    case ADDON_LOG_WARNING:
+      return "LOG_WARNING: ";
+    case ADDON_LOG_ERROR:
+      return "LOG_ERROR:   ";
+    case ADDON_LOG_SEVERE:
+      return "LOG_SEVERE:  ";
+    case ADDON_LOG_FATAL:
+      return "LOG_FATAL:   ";
+    default:
+      break;
+  }
+  return "LOG_UNKNOWN: ";
+}
+
+inline void kodiLog(const AddonLog logLevel, const char* format, ...)
```

Comment:
Has you added for later use? Seen the added log call is still by `kodi::Log`.

Has used this function on vfs.rar to make the debuging more easy, as I not need to look on Kodi log all the time.

Reply:
I forgot to remove it. Oops.

Reply:
Or maybe I will change it to this. It’s a good example of a custom logger.